packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -325,4 +325,32 @@ void main() {
 
     expect(textSpanTree, equals(expectedTextSpanTree));
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.android, TargetPlatform.iOS }));
+
+  testWidgets(
+    'buildTextSpanWithSpellCheckSuggestions corrects results when additions are made to misspelled word example',
```

Comment:
Nit: Maybe update this name?  Since it actually throws away the spell check suggestion now.